web/src/beta/features/PluginPlayground/Code/hook.ts

```diff
@@ -51,10 +51,69 @@ const getYmlJson = (file: FileType) => {
 export default ({ files }: Props) => {
   const [widgets, setWidgets] = useState<Widgets>();
   const [, setNotification] = useNotification();
+  const [fileOutputs, setFileOutputs] = useState<
+    {
+      title: string;
+      output: string;
+    }[]
+  >();
 
   const executeCode = useCallback(() => {
     const ymlFile = files.find((file) => file.title.endsWith(".yml"));
 
+    const jsFiles = files.filter((file) => file.title.endsWith(".js"));
+
+    const outputs = jsFiles.map((file) => {
+      const fn = new Function(
+        `"use strict";
+        const reearth = {
+          ui: {
+        show: function () {}
+          },
+          popup: {
+        show: function () {}
+          },
+        modal: {
+        show: function () {}
+          
+        };
```

Comment:
```suggestion
        const reearth = {
          ui: {
        show: function () {}
          },
          popup: {
        show: function () {}
          },
        modal: {
        show: function () {}
          }
        };
```
Missing one `}` here, it lead to error and stopped the code, maybe we should move this new Funtion into try catch as well?

Reply:
Thank you.
I fixed the brackets and thought it would be better to handle syntax errors with a try-catch block as well, so I made the corrections.

https://github.com/reearth/reearth-visualizer/pull/1263/commits/c8d3723d2391d9e96bf5b4ea21e18ca6dfd5b23d